src/ru/remanga/src/eu/kanade/tachiyomi/extension/ru/remanga/Remanga.kt

```diff
@@ -390,7 +390,8 @@ class Remanga : ConfigurableSource, HttpSource() {
                 altName = "Альтернативные названия:\n" + another_name + "\n\n"
             }
             val mediaNameLanguage = if (isEng.equals("rus")) en_name else rus_name
-            this.description = mediaNameLanguage + "\n" + ratingStar + " " + ratingValue + " (голосов: " + count_rating + ")\n" + altName + Jsoup.parse(o.description.replace("<p>", "").replace("</p>", "REPLACbR")).text().replace("REPLACbR", "\n")
+            this.description = mediaNameLanguage + "\n" + ratingStar + " " + ratingValue + " (голосов: " + count_rating + ")\n" + altName +
+                Jsoup.parse(o.description?.replace("<p>", "")?.replace("</p>", "REPLACbR") ?: "").text().replace("REPLACbR", "\n")
```

Comment:
In the current way, you're trying to parse an HTML string even when it's empty, and is not handling the case where a `p` tag might have some extra attributes. Try this instead:

```suggestion
                o.description
                    ?.let { Jsoup.parseBodyFragment(it) }
                    ?.select("p")
                    ?.joinToString("\n\n") { it.text() }
                    .orEmpty()
```